plugin/src/node-builder.ts

```diff
@@ -130,7 +130,7 @@ export function nodeBuilder(
         shopifyId: result.id,
         id: gatsbyApi.createNodeId(result.id),
```

Comment:
not sure if it applies here, but I would recommend prefixing `result.id` as well (if there is any chance same id is used in mulitple stores / instances). `Node.id` have to be unique, so when you create another node with same id you essentially remove previous one
```suggestion
        id: gatsbyApi.createNodeId(`${options.typeName}_${result.id}`),
```